### ISSUE TITLE
add CPLEX 12.9 to julia 1.1.1

### ISSUE THREAD

**Issue by lolitsbatman123** [JuliaOpt/CPLEX.jl#254]
So I have this text when i try to add cplex to julia
```
julia> Pkg.build("CPLEX")
  Building CPLEX → `C:\Users\Cheng\.julia\packages\CPLEX\FUtHu\deps\build.log`
┌ Error: Error building `CPLEX`:
│ ERROR: LoadError: UndefVarError: is_apple not defined
│ Stacktrace:
│  [1] top-level scope at none:0
│  [2] include at .\boot.jl:326 [inlined]
│  [3] include_relative(::Module, ::String) at .\loading.jl:1038
│  [4] include(::Module, ::String) at .\sysimg.jl:29
│  [5] include(::String) at .\client.jl:403
│  [6] top-level scope at none:0
│ in expression starting at C:\Users\Cheng\.julia\packages\CPLEX\FUtHu\deps\build.jl:18
```

**Comment by blegat** [comment]
Should be fixed in https://github.com/JuliaOpt/CPLEX.jl/pull/248/

**Comment by blegat** [comment]
Can you confirm that
```julia
] add CPLEX#master
] build CPLEX
```
fixes the issue ?

**Comment by odow** [comment]
Closed by the v0.6.0 tag: https://github.com/JuliaOpt/CPLEX.jl/releases/tag/v0.6.0.
